src/main/java/com/maxmind/geoip2/DatabaseProvider.java

```diff
@@ -59,6 +60,27 @@ AnonymousIpResponse anonymousIp(InetAddress ipAddress) throws IOException,
     Optional<AnonymousIpResponse> tryAnonymousIp(InetAddress ipAddress) throws IOException,
         GeoIp2Exception;
 
+    /**
+     * Look up an IP address in a GeoIP2 Anonymous Plus.
```

Comment:
In some spots we say `GeoIP Anonymous Plus` and in other spots we say `GeoIP2 Anonymous Plus`